pkg/configurator/client.go

```diff
@@ -276,6 +282,7 @@ func parseOSMConfigMap(configMap *v1.ConfigMap) *osmConfig {
 	osmConfigMap.ServiceCertValidityDuration, _ = GetStringValueForKey(configMap, serviceCertValidityDurationKey)
 	osmConfigMap.OutboundIPRangeExclusionList, _ = GetStringValueForKey(configMap, outboundIPRangeExclusionListKey)
 	osmConfigMap.EnablePrivilegedInitContainer, _ = GetBoolValueForKey(configMap, enablePrivilegedInitContainer)
+	osmConfigMap.EnableTicker, _ = GetStringValueForKey(configMap, enableTicker)
```

Comment:
It is time perhaps for us to create `GetDurationValueForKey` - we could use it for `ServiceCertValidityDuration` (a few lines above) as well

Reply:
yes, there's the other api that could make use of that too. Let me handle on separate commit though